frontend/packages/console-shared/src/components/formik-fields/RadioGroupField.scss

```
@@ -19,3 +19,9 @@
 
   }
 }
+
+.pf-c-radio {
+  &__description {
+    width: 100%;
+  }
+}
```

Move this styles definition into the scope of `ocs-radio-group-field` block, so it wont override the default pf-radio-button styles.
```suggestion

.pf-c-radio {
  &__description {
    width: 100%;
  }
 }
}
```